nucypher/blockchain/eth/sol/source/contracts/staking_contracts/PoolingStakingContractV2.sol

```diff
@@ -149,26 +175,28 @@ contract PoolingStakingContractV2 is InitializableStakingContract, Ownable {
     /**
      * @notice Get available reward in tokens for delegator
      */
-    function getAvailableReward(address _delegator)
-        public
-        view
-        returns (uint256)
-    {
+    function getAvailableDelegatorReward(address _delegator) public view returns (uint256) {
```

Comment:
Since we use `getAvailableDelegatorReward` here, should we match it with `getAvailableDelegatorETH` instead of `getAvailableETH`?

Reply:
sounds good

Reply:
✔️